src/monodomain_equations_routines.f90

```diff
@@ -1278,15 +1278,15 @@ SUBROUTINE MONODOMAIN_PRE_SOLVE(CONTROL_LOOP,SOLVER,ERR,ERROR,*)
     IF(ASSOCIATED(CONTROL_LOOP%PROBLEM)) THEN
```

Comment:
All right! I just checked one of my examples and it seems that none of the subroutines in monodomain_equation_routines.f90 is called. No idea why I was looking at that file...